plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependencyProvider.java

```diff
@@ -1467,61 +1484,49 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
             try {
-                File dir = new File(dependencyDir);
-
-                p = Runtime.getRuntime().exec("mvn install clean", null, dir);
-
-                BufferedReader stdInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
-
-                while ((stdInput.readLine()) != null) {
-                    if (monitor.isCanceled()) {
-                        p.destroy();
-                        Display.getDefault().syncExec(new Runnable() {
-                            public void run() {
-                                downloadSuccess = false;
-                                remove(dependencyDir + "/target");
-                                remove(dependencyDir + "/pom.xml");
-                                MessageDialog.openInformation(shell, "Cancelled",
-                                        "JDBC driver download operation was cancelled");
-                            }
-                        });
-                        throw new InterruptedException("JDBC driver download operation was cancelled");
-                    }
-                    monitor.worked(100);
+                MavenExecutionResult mvn = runMavenExecution(pomFile,
+                        Arrays.asList("org.apache.maven.plugins:maven-dependency-plugin:2.8:copy-dependencies"), null);
+                if (monitor.isCanceled()) {
+                    remove(dependencyDir + File.separator + "pom.xml");
+                    throw new InterruptedException("The download process was canceled");
                 }
 
-            } catch (Exception e) {
-                downloadSuccess = false;
-                e.printStackTrace();
+                if (mvn.hasExceptions()) {
+                    List<Throwable> rr = mvn.getExceptions();
+                    for (Throwable t : rr) {
+                        throw t;
+                    }
 
-                if (p == null) {
+                } else {
+                    remove(dependencyDir + File.separator + "pom.xml");
                     Display.getDefault().syncExec(new Runnable() {
                         public void run() {
-                            remove(dependencyDir + "/target");
-                            remove(dependencyDir + "/pom.xml");
-                            MessageDialog.openInformation(shell, "An Error has occoured",
-                                    "An error occurred when downloading the driver."
-                                            + "\nMake sure that you have installed Apache Maven.");
+                            MessageDialog.openInformation(shell, "Success",
+                                    "You have successfully downloaded the driver."
+                                            + "\n\nThe downloaded driver is located in the dependencies directory."
+                                            + "\nNote that downloaded drivers are not deployed with the CApp.");
```

Comment:
IMO we need to say "You need to copy jars on dependencies directory to <PRODUCT_HOME>/lib"

Reply:
fixed